### PR TITLE
Room creation and image attribute 

### PR DESCRIPTION
**Pull request by GeovanaRamos**
## Proposed Changes
Added room creation for admins and image field for each room.

## _Issue_ Related

https://github.com/SIGS-UnB/core/issues/232
https://github.com/SIGS-UnB/core/issues/233

## Type of Changes

- [X] New backend feature or update.
- [X] New frontend feature or update.
- [ ] Bug fix.
- [ ] Another change (what was the change?).

## Checklist

- [X] This Pull Request has a significant name.
- [X] The commits follow the [[commits policy]].
- [ ] The build is okay (tests, code climate).
- [X] This Pull Request mentions a related issue.
- [ ] The change was necessary to the progress of the project.

## Screenshots

newroom
showroom2
showrooom
indexrooms

### CODE REVIEW

**Comment by filypsdias** [comment]
How can we (@filypsdias & @CaioGabrielAraujo ) be assigned to this PR?

File: app/views/rooms/index.html.erb [hunk 1]
```diff
@@ -49,11 +56,12 @@
       <td></td>
       <td>
         <br>
-        <%= button_tag(type: 'submit', class: "btn pull-right btn-success") do %>
+        <%= button_tag(type: 'submit', class: "btn pull-right btn-success", style: "margin: 0px;") do %>
           <span class="fa fa-search"></span> Pesquisar
         <% end %>
       </td>
     </tr>
+
```
Comment (arthurbdiniz) [inline app/views/rooms/index.html.erb]: No extra line here

File: app/models/room.rb [hunk 2]
```diff
@@ -45,4 +45,6 @@ class Room < ApplicationRecord
                             message: MAXIMUM_FOR_THE_CAPACITY_EXCEPTION
 
   self.per_page = 10
+
+  has_one_attached :photo
```
Comment (arthurbdiniz) [inline app/models/room.rb]: This statement should be on the top with:

```ruby
class Room < ApplicationRecord
  belongs_to :building
  has_many :allocations, dependent: :destroy
  belongs_to :department
  has_and_belongs_to_many :category
  belongs_to :department
```

File: app/views/rooms/index.html.erb (hunk 1 above)
Comment (arthurbdiniz) [inline app/views/rooms/index.html.erb]: No extra line here

**Review by arthurbdiniz — CHANGES_REQUESTED** [review]
Just some style changes to do here

**Comment by GeovanaRamos** [comment]
@arthurbdiniz a test that was passing now is broken for no reason. Any ideias? I only removed a blank line and changed the attribute line.

image

**Comment by GeovanaRamos** [comment]
Found a test that sometimes was breaking and sometimes was not! It's all good now.

**Review by arthurbdiniz — APPROVED** [review]

**Pull request merged**